lib/onelogin/ruby-saml/saml_message.rb

```diff
@@ -73,11 +75,11 @@ def encode(encoded)
       ##
       # Check if +string+ is base64 encoded
       #
-      # The function is not strict and does allow newline. This is because some SAML implementations
-      # uses newline in the base64-encoded data, even if they shouldn't have (RFC4648).
-      def is_base64?(string)
-        string = string.gsub(/\r\n/, "").gsub(/\n/, "")
-        string.match(%r{\A(([A-Za-z0-9+/]{4}))*([A-Za-z0-9+/]{4}|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==)\Z})
+      # The function is not strict and allows newlines. This is because some
+      # SAML implementations use newlines in the base64-encoded data even if
+      # they shouldn't (RFC4648).
+      def base64_formatted?(string)
```

Comment:
can you use YARD style comments here and explicitly document the return value as well?